### DISCUSSION TITLE
How to watch clipboard changes?

### DISCUSSION THREAD

**Discussion by jet10000** [flet-dev/flet#336] [General]
I want to get clipboard screenshot image, How to watch clipboard images changes?

**Comment by FeodorFitsner** [comment]
Not sure about "watching" - all examples for Flutter I've seen so far are just polling clipboard.

Regarding image fetching from a clipboard it could be implemented using this Flutter package: https://pub.dev/packages/pasteboard

**Reply by FeodorFitsner** [reply]
Filed a new issue: https://github.com/flet-dev/flet/issues/349

**Comment by jet10000** [comment]
The function of polling clipboard has not been added yet.

**Comment by jet10000** [comment]
I now get the pasteboard image via a button.
